src/transformers/pipelines.py

```diff
@@ -420,7 +426,7 @@ def inputs_for_model(self, features: Union[dict, List[dict]]) -> Dict:
         """
         args = ["input_ids", "attention_mask"]
 
-        if not isinstance(self.model.config, (DistilBertConfig, XLMConfig, RobertaConfig, BartConfig)):
+        if not isinstance(self.model.config, (DistilBertConfig, XLMConfig, RobertaConfig, BartConfig, T5Config)):
```

Comment:
Ok, note that we can now remove the `inputs_for_model` from pipelines since #3116 
Let's do it in another PR cleaning up pipelines later.